lib/mix/tasks/generate_alter_table_migrations.ex

```diff
@@ -1,33 +1,32 @@
 defmodule Mix.Tasks.RailwayIpc.GenerateAlterTableMigrations do
+  @moduledoc """
```

Comment:
Adding a module doc here so that a dev can run a command to figure out how to use this:
```
[17:56:48] (add-railway-ui) registrar_umbrella
// ♥ mix help railway_ipc.generate_migrations

                      mix railway_ipc.generate_migrations

Mix task for generating Railway migration files

Run with mix railway_ipc.generate_migrations ./path/to/migrations If not path
is passed in, the task will default to ./priv/repo/migrations

Location: _build/dev/lib/railway_ipc/ebin
```